cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/AppFabricServerTest.java

```diff
@@ -105,4 +97,24 @@ public EndpointStrategy get() {
     socket.startHandshake();
     appFabricServer.stopAndWait();
   }
+
+  private Injector getInjector(CConfiguration cConf, @Nullable SConfiguration sConf) {
```

Comment:
This method is doing more than returning injector, maybe break this method down?

Reply:
Creating the injector is just one line, so I don't break them into separate methods. I renamed the method to make it more descriptive.